client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -22,9 +22,12 @@
    private final Sequence startingPosition;
    @Getter
    private final long groupRefreshTimeMillis;
+   @Getter
+   private final long automaticCheckpointIntervalMillis;   
 
    public static final class ReaderGroupConfigBuilder {
-       long groupRefreshTimeMillis = 3000;
+       private long groupRefreshTimeMillis = 3000;
+       private long automaticCheckpointIntervalMillis = -1;
```

Comment:
Although I think I have abused of the term timeout, my point still stands. If I don't care about checkpoints, why would I have to worry about setting an automatic checkpoint interval? 

I see two choices here:

1- The parameter is `enableCheckpointing, which is false by default and we hardwire the interval.
2- We change the parameter name to something like `rebalanceInterval` because I don't care about checkpoints in this scenario by I care about rebalances.